indexers/staking-squid/src/events/bundle.ts

```diff
@@ -14,8 +14,8 @@ export async function processBundleStoredEvent(
   const domainId = Number(event.args.domainId);
   const operatorId = Number(event.args.bundleAuthor);
   const lastDomainBlockNumber = Number(
-    extrinsic.call?.args[0].value.sealed_header.header.receipt
-      .domain_block_number
+    extrinsic.call?.args.opaqueBundle.sealedHeader.header.receipt
```

Comment:
I think it would be more efficient get this out of storage on each block rather than from each bundle. `domains.headDomainNumber(domainID)`